packages/frontier-plugins/plugin-shared/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@rdfrontier/plugin-shared",
+  "version": "0.0.1",
+  "description": "plugin shared functionalities",
+  "license": "UNLICENSED",
+  "author": "Realdecoy",
+  "main": "dist/index.js",
+  "module": "module/index.js",
+  "files": [
+    "dist/",
+    "module/",
+    "types/"
+  ],
+  "scripts": {
```

Comment:
This package has no dependencies but still references dependencies in code and in scripts?

Reply:
Updated, mistake on my part